Tests/UnitTests/Core/BrowserInfoTests.swift

```diff
@@ -15,7 +15,7 @@ class BrowserInfoTests: XCTestCase {
             XCTAssertNotNil(info?.userAgent)
             browserInfoExpectation.fulfill()
         }
-        waitForExpectations(timeout: 10, handler: nil)
+        waitForExpectations(timeout: 120, handler: nil)
```

Comment:
Why was this increased?

Reply:
New xcode/MacOS combination takes longer for WKWebKit javascript engine to warmup on CI